packages/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -348,15 +352,16 @@ def regular_file_cud(folder, log_monitor, time_travel=False, n_regular=1, min_ti
     :type folder: String
     :param log_monitor: File event monitor
     :type log_monitor: FileMonitor
+    :param file_list: List/Dict with the file names and content.
+    List -> ['name0', 'name1'] -- Dict -> {'name0': 'content0', 'name1': 'content1'}
```

Comment:
Lacks the specification of default content in case of using a list

Reply:
Done.